diagnostic.md

```diff
@@ -6,70 +6,72 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books', to: 'books#index'
+resources :books, only: [:index]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+Creates tables based on the migration files in your project - acts like a template for your DB.
```

Comment:
Not only creates tables but it executes any code found within the Active Record migration files in db/migrate. The migrations make it so that we don't have to modify our database directly.